blocks/header/header.css

```diff
@@ -1,51 +1,36 @@
+:root {
```

Comment:
If these variables are scoped within header I would move it inside another name other than `root`.